docs/operation_center/integration_catalog/cloud_and_saas/cloudflare/cloudflare.md

```diff
@@ -25,12 +25,40 @@ To do so, connect to [Cloudflare Console](https://dash.cloudflare.com/) to colle
 - `Cloudflare Zone ID`
 - `Cloudflare account email`
 
+#### with logpush
 
-You now have all information to configure the Cloudflare module and its `Send events` action to SEKOIA.IO.
+!!! note
+
+    It's the recommended way to get your events from Cloudflare.
+
+Configure a [logpull job](https://developers.cloudflare.com/logs/reference/logpush-api-configuration/) with the following destination:
+
+`https://intake.sekoia.io/plain/batch?header_X-SEKOIAIO-INTAKE-KEY=<your intake key>`
+
+
+##### Manage logpush with cURL
+
+```bash
+$ curl -X POST https://api.cloudflare.com/client/v4/zones/<CLOUDFLARE_ZONE_ID>/logpush/jobs \
+-H "X-Auth-Email: <CLOUDFLARE_EMAIL>" \
+-H "X-Auth-Key: <CLOUDFLARE_API_KEY>" \
+-H "Content-Type: application/json" \
+--data '{
+    "dataset": "http_requests",
+    "enabled": true,
+    "max_upload_bytes": 5000000,
+    "max_upload_records": 1000,
+    "destination_conf": "https://intake.sekoia.io/plain/batch?header_X-SEKOIAIO-INTAKE-KEY=<INTAKE_KEY>"
+}'
+```
+
+
+#### with logpull
 
 You can now enable Cloudflare log retention (see [associated documentation](https://developers.cloudflare.com/logs/logpull/enabling-log-retention/)). 
```

Comment:
I didn't get why this section start with this sentence, maybe you deleted something ?

Reply:
This phrase `You now have all information to configure the Cloudflare module and its `Send events` action to SEKOIA.IO.` was deleted